Program.cs

```diff
@@ -24,38 +25,60 @@ static void Main(string[] args)
             List<string> SupportedExtensions = new List<string>
             {
                 ".mp3",
+                ".flac",
```

Comment:
Surprisingly, `.flac` files seem to have roughly the same amount of metadata as `.mp3` files do, so we fully support them.